### PR TITLE
add basic login

### PR DESCRIPTION
**Pull request by Sail338**
1.) Endpoint to call authorize and read endpoints and check if the person is a director

### CODE REVIEW

File: login.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            profile = requests.post(BASE_URL + "/read",json=(profile_request_data_dict))
+            #for some reason the query returns a json array?
+            profile_json = profile.json()
+            profile_body = profile_json["body"][0]
+            #if they arent a director yeet them out
+            if profile_body['role']['director'] == False:
+                return {"statusCode":400,"body":"You are not a director, please contact a director for updates"}
+            else:
+                return {statusCode:200,"body":json.dumps({"email":resp_json["email"],"token":resp_json["token"]})}
+        else:
```
Comment (hemangandhi) [inline login.py:31]: I recommend just forwarding the LCS error messages, in case they turn out to be more informative.

**Review by hemangandhi — APPROVED** [review]
Minor nitpick, but this is gonna make one heck of an example for how to use LCS.

Thx.

File: login.py (hunk 1 above)
Comment (Sail338) [reply login.py:31]: would LCS return an error if a non director logs in?  should VHX scheduler allow non directors to login?

Comment (hemangandhi) [reply login.py:31]: It's more about the user not existing etc.

Comment (Sail338) [reply login.py:31]: got it ok, if LCS returns a 400 ill forward the error

**Pull request merged**